examples/AWSDriverExample/src/main/java/software/amazon/AwsIamAuthenticationMariadbExample.java

```diff
@@ -0,0 +1,57 @@
+/*
+ *
+ *     Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *
+ *     Licensed under the Apache License, Version 2.0 (the "License").
+ *     You may not use this file except in compliance with the License.
+ *     You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *     Unless required by applicable law or agreed to in writing, software
+ *     distributed under the License is distributed on an "AS IS" BASIS,
+ *     WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *     See the License for the specific language governing permissions and
+ *     limitations under the License.
+ *
+ */
+
+package software.amazon;
+
+import java.sql.Connection;
+import java.sql.DriverManager;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.sql.Statement;
+import java.util.Properties;
+import software.amazon.jdbc.PropertyDefinition;
+
+public class AwsIamAuthenticationMariadbExample {
+  public static final String MYSQL_CONNECTION_STRING =
+      "jdbc:aws-wrapper:mysql://db-identifier.cluster-XYZ.us-east-2.rds.amazonaws.com:3306?permitMysqlScheme";
```

Comment:
is permitMysqlScheme required?
Since this is a mariaDB example why not just use the `mariadb` protocol

Reply:
I think this parameter is required because Aurora only supports mysql and postgresql, so we only can use mariadb as a driver to connect to mysql

Reply:
I just did it this way because we tested it like this, we don't have tests for iam authentication connecting to a mariadb instance